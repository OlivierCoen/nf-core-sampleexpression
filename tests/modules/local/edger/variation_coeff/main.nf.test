nextflow_process {

    name "Test Process EDGER_VARIATIONCOEFF"
    script "modules/local/edger/variation_coeff/main.nf"
    process "EDGER_VARIATIONCOEFF"
    tag "edger"

    test("Should run without failures") {

        when {

            process {
                """
                input[0] = file('$baseDir/tests/input/deseq2/variation_coeff/all_counts.csv')
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
