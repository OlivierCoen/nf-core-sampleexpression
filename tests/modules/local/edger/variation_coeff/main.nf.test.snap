{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "all_counts_normalized.csv:md5,008ad1d4afd84152185753cd1e0ca5ed"
                ],
                "csv": [
                    "all_counts_normalized.csv:md5,008ad1d4afd84152185753cd1e0ca5ed"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.04.4"
        },
        "timestamp": "2024-10-16T19:24:47.807739457"
    }
}